cli/testflinger_cli/__init__.py

```diff
@@ -496,6 +524,143 @@ def queue_status(self):
             )
         print(output)
 
+    def _queue_status_jobs(self):
+        """Show the status of jobs in a specified queue (new functionality)."""
```

Comment:
This can be reformatted as follow based on current suggestions:
```suggestion
    def _queue_status_jobs(self):
        """Retrieve the status of jobs in a specified queue"""
```